File: 프로그래머스/1/131705. 삼총사/삼총사.py
def solution(number):
    n = len(number)
    answer = 0
    
    for a in range(n - 2):
        for b in range(a + 1, n - 1):
            for c in range(b + 1, n):
                if not number[a] + number[b] + number[c]:
                    answer += 1
    
    return answer

File: 프로그래머스/1/131705. 삼총사/삼총사.java
class Solution {
    public int solution(int[] number) {
        int answer = 0;
        int length = number.length;
        
        for (int a=0; a<length-2; a++)
            for (int b=a+1; b<length-1; b++)
                for (int c=b+1; c<length; c++)
                    if (number[a] + number[b] + number[c] == 0)
                        answer += 1;
        
        return answer;
    }
}